Snowflake.Data.Tests/Snowflake.Data.Tests.csproj

```diff
@@ -11,6 +11,9 @@
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <LangVersion>9</LangVersion>
     <DefineConstants>$(SEQUENTIAL_ENV)</DefineConstants>
+    <AssemblyOriginatorKeyFile>..\sign\publicKey.snk</AssemblyOriginatorKeyFile>

```

Comment:
Snowflake.Data.Tests is not released so we don't need to sign it.

Reply:
The sign is required to use the InternalVisibleTo, so it's only signed with a delay-sign for local development. @sfc-gh-knozderko